.gitignore

```diff
@@ -2,7 +2,7 @@
 .*.swp
 intervals/
 __pycache__
-results/*
+results*/*
```

Comment:
curious if this * was required?

Reply:
I don't think it should be `results*/*`-- it was working just fine as is

Reply:
no ... I was running diff configs and renaming the results files so as not to overwrite. This was to prevent them uploading.